main.js

```diff
@@ -80,6 +80,17 @@ app.on('ready', () => {
       Menu.setApplicationMenu(menu)
     }
 
+    const inputMenu = Menu.buildFromTemplate([
```

Comment:
per our slack discussion, please change to only allow for 'paste'. thanks and look forward to working with you on the wallet :)

Reply:
@dvdschwrtz done 👌